src/linagora.esn.unifiedinbox/app/services/mailboxes-filter/mailboxes-filter-service.js

```diff
@@ -200,7 +200,10 @@ angular.module('linagora.esn.unifiedinbox')
     }
 
     function _setFiltersOnServer(filters) {
-      return asyncJmapAction({ success: 'Filters set', failure: 'Error setting filters' }, function(client) {
+      return asyncJmapAction({
+        success: esnI18nService.translate('Filters set'),
+        failure: 'Error setting filters'
```

Comment:
why the failure is not translated?

Reply:
@oubchid : please also add translation for "Error setting filters"

Reply:
done @MichaelBailly @rouazana